frontend/packages/console-app/src/hooks/useCSPVioliationDetector.tsx

```diff
@@ -28,34 +29,22 @@ export const newCSPViolationReport = (
   event: SecurityPolicyViolationEvent,
 ): CSPViolationReport => ({
   ..._.pick(event, [
-    // The URI of the resource that was blocked because it violates a policy.
     'blockedURI',
-    // The column number in the document or worker at which the violation occurred.
     'columnNumber',
-    // Whether the user agent is configured to enforce or just report the policy violation.
     'disposition',
-    // The URI of the document or worker in which the violation occurred.
     'documentURI',
-    // The directive that was violated.
     'effectiveDirective',
-    // The line number in the document or worker at which the violation occurred.
     'lineNumber',
-    // The policy whose enforcement caused the violation.
     'originalPolicy',
-    // The URL for the referrer of the resources whose policy was violated, or null.
     'referrer',
-    // A sample of the resource that caused the violation, usually the first 40 characters.
-    // This will only be populated if the resource is an inline script, event handler or style.
     'sample',
-    // If the violation occurred as a result of a script, this will be the URL of the script.
     'sourceFile',
-    // HTTP status code of the document or worker in which the violation occurred.
     'statusCode',
   ]),
   pluginName: pluginName || 'none',
```

Comment:
What if there's a plugin named `none`? :smiley: 

Wouldn't it be better to use `null` or empty string here, as opposed to some magic string?